guide/getting-started/visual-language.md

```diff
@@ -36,10 +36,13 @@ layout = "full-width"
 
 # The visual language of bitcoin
 
-Since Bitcoin does not have a central authority, there is no specific symbol or style that we can consider official. However, there is a large body of community art and design work, along with a few proposals that ended up being highly influential. Let's start with some key moments that led us to the visual that many of us associate with Bitcoin today.
+Since Bitcoin answers to no central authority, there is no single symbol or logo that can visually represent it, though many try. Here are some of the most recognized attempts to create a universal logo for Bitcoin.
+
 
 ### Original symbol
 
+Satoshi Nakamoto created this lettered golden coin for the original Bitcoin Core client.
```

Comment:
Layout got messed up. Move this text to where the previous text was removed (between the image include and the closing div).

Reply:
Nice catch.

Fixed https://github.com/BitcoinDesign/Guide/pull/356/commits/9906216a0a0dc48b3b902510c2a40e583ad95657

![Screenshot 2021-05-27 at 17 20 34](https://user-images.githubusercontent.com/36959754/119852896-00478080-bf10-11eb-9929-894ab64c52d1.png)